shared/metrics/src/main/java/io/pravega/shared/metrics/MetricsConfig.java

```diff
@@ -96,6 +101,30 @@
     @Getter
     private final int statsDPort;
 
+    /**
+     * The URI of InfluxDB endpoint.
+     */
+    @Getter
+    private final String influxDBUri;
+
+    /**
+     * The name of InfluxDB.
+     */
+    @Getter
+    private final String influxDBName;
+
+    /**
+     * The username to access InfluxDB.
+     */
+    @Getter
+    private final String influxDBUserName;
+
+    /**
+     * The password of user account accessing InfluxDB.
+     */
+    @Getter
+    private final String influxDBPassword;
```

Comment:
Why would this be here?

Reply:
In case Pravega sends metric directly to InfluxDB, username and password are needed to access the database.